utils/rpms/daos.spec

```diff
@@ -278,6 +281,17 @@ Requires: hdf5
 This is the package needed to use the DAOS serialization and deserialization
 tools, as well as the preserve option for the filesystem copy tool.
 
+%package mofed_shim
```

Comment:
if we create a new RPM for this, can we prefix that RPM with "daos", e.g. daos-mofed-shim?

Reply:
It will be.  `%package` without a `-n` argument prefixes the base package name in front of sub-packages' names.  From the daos package installation in the Jenkins run:

```
[2021-11-26T22:53:30.500Z]  daos-client-tests             x86_64  2.1.100-9.7435.g2c055fb4.el8  build.hpdd.intel.com_job_daos-stack_job_daos_job_PR-7481_lastSuccessfulBuild_artifact_artifacts_centos8_  3.1 M
[2021-11-26T22:53:30.500Z]  daos-client-tests-openmpi     x86_64  2.1.100-9.7435.g2c055fb4.el8  build.hpdd.intel.com_job_daos-stack_job_daos_job_PR-7481_lastSuccessfulBuild_artifact_artifacts_centos8_  468 k
[2021-11-26T22:53:30.500Z]  daos-mofed_shim               x86_64  2.1.100-9.7435.g2c055fb4.el8  build.hpdd.intel.com_job_daos-stack_job_daos_job_PR-7481_lastSuccessfulBuild_artifact_artifacts_centos8_   22 k
[2021-11-26T22:53:30.500Z]  daos-server-tests             x86_64  2.1.100-9.7435.g2c055fb4.el8  build.hpdd.intel.com_job_daos-stack_job_daos_job_PR-7481_lastSuccessfulBuild_artifact_artifacts_centos8_  455 k
[2021-11-26T22:53:30.500Z]  daos-server-tests-openmpi     x86_64  2.1.100-9.7435.g2c055fb4.el8  build.hpdd.intel.com_job_daos-stack_job_daos_job_PR-7481_lastSuccessfulBuild_artifact_artifacts_centos8_  168 k
```

Reply:
ok thanks, i wasn't aware of that. 
i would still change the underscore to a dash (daos-mofed-shim), as it's always a pain to switch between the two when typing package names.